src/backend/access/table/table.c

```diff
@@ -249,11 +249,6 @@ CdbTryOpenTable(Oid relid, LOCKMODE reqmode, bool *lockUpgraded)
 		lockmode = reqmode;
```

Comment:
Need more commit messages in current commit. Because current PR won't be squash when merged.